scenario-compiler/src/test/scala/pl/touk/nussknacker/engine/InterpreterSpec.scala

```diff
@@ -196,7 +196,8 @@ class InterpreterSpec extends AnyFunSuite with Matchers {
           ComponentDefinitionExtractionMode.FinalDefinition
         ),
       ModelDefinitionBuilder.emptyExpressionConfig,
-      ClassExtractionSettings.Default
```

Comment:
Where are tests showing how our runtime behave when there is no sink? Does our scenario builder is ready for such construction of scenario?

Reply:
Hmm interesting, I haven't seen it, now I see it. The approach with the specific engine is fine. The only thing that I'm worry about is that the name of test suggest that you are testing the listener and in fact you wanted to test scenario compilation process. We have `FlinkTestScenarioRunner`, which we use for different scenario combinations tests. Can you reuse it?

Reply:
I refactored it - there are now 2 separate test files. The new on e contains only tests for scenarios without sinks. There are also assertions of compilation errors (in situations, when ending with sink is required). I use the flinkMiniCluster, but not the  FlinkTestScenarioRunner - it is much more complex (which is not required here) but does not allow the customization (custom config substitutions, multiple custom sources) that is required here.